tensorflow/python/ops/embedding_ops.py

```diff
@@ -307,6 +307,12 @@ def embedding_lookup(
       element must be appropriately sized for the given `partition_strategy`.
     ids: A `Tensor` or a 'RaggedTensor' with type `int32` or `int64` containing
       the ids to be looked up in `params`.
+      
+      Caution: Without XLA,on CPU, if an out of bound id is found, an error is raised.
+      On GPU, if an out of bound id is found, a 0 is stored in the corresponding
+      output value.With XLA, for both CPU and GPU, all out of bound ids are
```

Comment:
missing space in "value.With XLA"

Though I suggest steering clear of specifying explicitly what happens in XLA vs non-XLA, CPU vs GPU since these may change over time, and the behavior depends on _different_ ops, not this one specifically in embedding lookup.  Instead, I suggest simply stating that out-of-bounds indices will result in undefined behavior, which will differ between devices and backends.

Reply:
That means is this caution note is suffice?

**Caution**: Out-of-bounds indices will result in undefined behavior, which will differ between devices and backends.

Reply:
Yes

Reply:
Done the changes as suugested.